css_tasks/task4/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Task 4</title>
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <section class = "schedule">
+      <h3 class="schedule__header">Weekly Schedule</h3>
+      <input type="checkbox" id="input--leisure" class="schedule__input--leisure" />
```

Comment:
FYI: if your classes is bem modifiers you can't use them without base class, i meat it shoud be `class="schedule__input schedule__input--leisure"`

and this id-s should be written with one `-` as `input-leisure` they for your `label` tag and have not connection with your styles

Reply:
Classes naming has been refactored